src/v2/Components/Seo/LocalBusiness.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react"
+import { StructuredData } from "./StructuredData"
+import _ from "underscore"
+
+interface PostalAddressSchemaData {
+  "@type": string
+  addressCountry: string
+  addressLocality: string
+  addressRegion: string
+  postalCode: string
+  streetAddress: string
+  telephone: string
+}
```

Comment:
I know we can import a package of these types instead of manually doing them. :looking:

Reply:
https://www.npmjs.com/package/schema-dts

Reply:
I've used this before and it really takes the guess work out of it, and types don't add any weight to the bundle since they are discarded

Reply:
Nice one, will add! 🙌

Reply:
More chatter here but I had trouble adding this package and then using it for the `PostalAddress` data:

https://artsy.slack.com/archives/CP9P4KR35/p1628277103099800

Going to bail on this and can return to this topic later if it's warranted - hope that's ok!

Reply:
Yah is fine. FWIW looking at this now — you wouldn't use `PostalAddress` directly but rather the entire `LocalBusiness` type. If we ever want to do JSON+LD in the future we should definitely just use the provided schema. I suspect that these will be invalid when there's some nulls around, etc.